### PR TITLE
Fixes #162 Why are the values of ODE state variables in [0 .. AbsTol]…

### LINKED ISSUE #162: Why are the values of ODE state variables in [0 .. AbsTol] not set to 0 ?
According to the comment in the code, the values of ODE variables in the range `[-AbsTol .. AbsTol]` are set to zero:
https://github.com/Open-Systems-Pharmacology/OSPSuite.SimModel/blob/7092dd785819f9904abbc1f40cc33c8e4be05a30/src/OSPSuite.SimModelNative/src/DESolver.cpp#L350-L364

However, when looking at the implementation: only the values in `[-AbsTol .. 0]` are set to zero, but **not the values in** `[0 .. AbsTol]`
https://github.com/Open-Systems-Pharmacology/OSPSuite.SimModel/blob/7092dd785819f9904abbc1f40cc33c8e4be05a30/src/OSPSuite.SimModelNative/src/SimulationTask.cpp#L186-L193

I assume originally the values in `[0 .. AbsTol]` were set to zero as well, but we got some kind of problems with this (but I don't remember which exactly.). 

_Originally posted by @Yuri05 in https://github.com/Open-Systems-Pharmacology/OSPSuite-R/pull/1341#discussion_r1457388258_

### PR DESCRIPTION
**Pull request by Yuri05**
… not set to 0

### CODE REVIEW

File: src/OSPSuite.SimModelNative/src/SimulationTask.cpp [hunk 1]
```diff
@@ -187,7 +187,7 @@ void SimulationTask::SetValuesBelowAbsTolLevelToZero(double * values, int values
 {
 	for(int i =0; i<valuesSize;i++)
 	{
-		if((values[i] < 0.0) && (values[i] > -absTol))
+		if (fabs(values[i]) < absTol)
 			values[i] = 0.0;
```
Comment (Yuri05) [inline src/OSPSuite.SimModelNative/src/SimulationTask.cpp:191]: now setting the values in `(-AbsTol .. AbsTol)` to zero.

Comment (msevestre) [reply src/OSPSuite.SimModelNative/src/SimulationTask.cpp:191]: Yeah

Comment (msevestre) [reply src/OSPSuite.SimModelNative/src/SimulationTask.cpp:191]: The conditions previously was so hard to read lol

File: tests/TestData/RemoveBelowAbsTolTest.xml
Comment (Yuri05) [inline tests/TestData/RemoveBelowAbsTolTest.xml:1]: Test simulation with AbsTol=1E-9
Thus the first part of the lower curve will be set to 0 now
{C23D5BA1-A66C-4361-B891-5822C79F491E}

**Comment by Yuri05** [comment]
hmm, on AppVeyor 1 test fails now (locally all tests run through).
Also interesting: in the overview, all tests are marked as successful

```
Appveyor.TestLogger: 77 test results reported (77 enqueued).
Test Run Failed.
Total tests: 77
     Passed: 76
    Skipped: 1
```

File: tests/TestData/ODM1.xml
Comment (Yuri05) [inline tests/TestData/ODM1.xml:1]: test file was not used in any test case

**Review by rwmcintosh — APPROVED** [review]
Do we still need `Previous Visual Studio 2022`

**Comment by Yuri05** [comment]
> Do we still need `Previous Visual Studio 2022`

I think no. Should be roll backed

**Pull request merged**